package.json

```diff
@@ -72,6 +72,36 @@
         "aliases": [
           "Go"
         ]
+      },
+      {
+        "id": "go.mod",
+        "extensions": [
+          "go.mod"
+        ],
+        "aliases": [
+          "Go Module File"
+        ]
+      },
+      {
+        "id": "go.sum",
+        "extensions": [
+          "go.sum"
+        ],
+        "aliases": [
+          "Go Sum File"
```

Comment:
Can we call this `Go Checksum File`? It's a minor nitpick.

Reply:
Yep. Good idea.